pypuf/experiments/experimenter.py

```diff
@@ -239,6 +246,12 @@ def update_status_error(exception):
 
             # experiment execution
             for experiment in experiments:
+                # Assign experiment to GPU (if used) : might be replaced by more sophisticated load balancer
+                if self.gpu_limit > 0:
+                    gpu_num = self.gpu_counter % self.gpu_limit
```

Comment:
Will this assign multiple experiments to the same GPU? Or am I required to set `self.cpu_limit = self.gpu_limit`? In the later case, it may make sense to rename `cpu_limit` to `process_limit`.

Reply:
It will definitely assign multiple processes to one GPU if cpu_limit > gpu_limit.

Reply:
I propose to check during experimenter initialization if `gpu_limit is not None and cpu_limit > gpu_limit` and throw an error if so. Unless there is a valid use case where assigning multiple experiments to one GPU makes sense.

Reply:
Tensorflow is able to leverage CUDAs multi-processing server, so using one GPU for multiple experiments at the same time makes sense for performance improvements.

Reply:
Alrighty, then we shall leave it as is